Post/views.py

```diff
@@ -11,8 +12,20 @@ class PostListView(ListView):
     context_object_name = 'posts'
 
 
+@login_required
+def AddAPost(request):
+    form = AddAPostForm(request.POST)
+    if form.is_valid():
+        form.save()
+        return redirect("http://127.0.0.1:8000/postList")
```

Comment:
redirect("http://127.0.0.1:8000/postList")
Will not work in production, some think like that
return HttpResponseRedirect('postList?')

Reply:
I fixed it.